tests/backward_compatibility_tests.sh

```diff
@@ -35,15 +35,17 @@ rm -r  ~/.sky/wheels || true
 cd ../sky-master
 git pull origin master
 pip uninstall -y skypilot
-pip install -e ".[all]"
+pip install uv
+uv pip install -e ".[all]"
```

Comment:
Could you test this with a completely empty conda base environment? I think installation of the azure-cli dependency will fail. See #4414 about that bug.

Reply:
Ah, I see. What about now? Install `azure-cli` beforehand, the same as in the gitHub workflow configuration.